src/components/PatchesPage/index.tsx

```diff
@@ -70,13 +77,24 @@ export const PatchesPage: React.FC<Props> = ({
   });
   usePageTitle(pageTitle);
 
-  const onCheckboxChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+  const commitQueueCheckboxOnChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ): void => {
     setIsCommitQueueCheckboxChecked(e.target.checked);
     Cookies.set(cookie, e.target.checked ? "true" : "false");
     // eslint-disable-next-line no-unused-expressions
     analyticsObject.sendEvent({ name: "Filter Commit Queue" });
   };
 
+  const includeHiddenCheckboxOnChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ): void => {
+    setIsIncludeHiddenCheckboxChecked(e.target.checked);
+    Cookies.set(INCLUDE_HIDDEN_PATCHES, e.target.checked ? "true" : "false");
```

Comment:
When I load this page with this cookie set to true, the checkbox was checked but hidden patches are not visible in the list (i.e. the state is set but the query does not include the parameter).

Tbh I don't know how helpful a cookie is for this setting. Unlike the commit queue, it's probably something that would be looked at in special circumstances, not every time a user visits this page. But if you think it's worth including then the bug should be fixed!